ONSdigital/response-operations-ui
# Update manage user accounts

# What and why?
The Manage use accounts and manage account pages need updating to improve usability. This PR separates the edit and delete page functionality from the manage user accounts page along with some updates to the delete user account page and manage account page. 
# How to test?
Go through the edit user account journey, including deleting an account and editing permissions on an exisiting account to ensure all buttons function as expected and redirect correctly. 
# Jira
https://jira.ons.gov.uk/secure/RapidBoard.jspa?rapidView=1493&projectKey=RAS&view=detail&selectedIssue=RAS-271
